README.md

```diff
@@ -42,6 +42,45 @@ Explore our [collaborative examples](https://github.com/drifting-in-space/y-swee
 
 All examples are open source and live in this repository, within [/examples](https://github.com/drifting-in-space/y-sweet/tree/main/examples).
 
+## Usage
+
+Check the [vanilla js example](/examples/vanilla/) for more details.
+
+### On the server
+``` js
+import { createDoc } from '@y-sweet/sdk'
+// Get the client token from the y-sweet server. This is basically the user's "password" to edit the "myDoc123" doc.
+app.get('/client-token', async (req, res) => {
+  const clientToken = await createDoc('myDoc123')
+  res.send(clientToken)
+})
+```
+
+### On the client
+``` js
+import * as Y from 'yjs';
+import { createYjsProvider } from '@y-sweet/client';
+
+// First, fetch a client token that can access the docId in the URL.
+const clientToken = await fetch("http://localhost:9090/client-token").then(r => r.json())
```

Comment:
I kinda think we should leave this out and just have a comment down at `createYjsProvider()` that says something like "this is the clientToken that is generated above in the server code and must be passed to the client". Thoughts?

Reply:
i think we could probably go without explaining this too since it should probably be implied that the client token would come from the server